app/models/content/resource.rb

```diff
@@ -61,4 +66,18 @@ def annotated_paragraphs editable: false
   def title
     super || resource.title
   end
+
+  def footnote_annotations
```

Comment:
This is not specific to this code review, but I have a general suggestion. What if you pick a commenting style and start adding comments for new code?

I like [the policy here](https://github.com/airbnb/ruby#commenting) in AirBnB's style guide, which basically recommends a comment describing the purpose of each class, a comment describing the purpose and input/output of each function, and a comment describing the purpose of any complicated block of code. That's a lot like what I do in CAP -- [here's an example function](https://github.com/jcushman/capstone/blob/5c39f23f6b723dc2f923795ae87ac40ac37c1fed/capstone/scripts/make_pdf.py#L49).

The idea here is if you know the few lines of code that are causing a bug, it's usually obvious how to fix it. But if you have 200 lines of code you haven't looked at in a year, you don't want to slog through and figure out the point of them all in order to find the few lines that matter to the bug you're fixing. You want section headings to scan through.

So here I might add a comment before the function like

```
# Get string representation of note and link annotations, 
# formatted like "* Note 1 ** Note 2 *** Note 3 ".
```

When I do this I tend to do it as the very last step before sending a pull request, with only minimal commenting as I go along, because I don't want to rewrite stuff as I mess around. But then before committing, I go through and make sure that the new code is sorted in blocks that make sense, variable names still make sense (because often the meaning changes as I go along), and each function or block has a short comment that will explain what it's for when I come back in a year.

Totally up to you how much of that is helpful -- but my hunch is that adding some block-level comments would pay off as you go along.